.github/workflows/bintray_upload.yml

```diff
@@ -0,0 +1,31 @@
+name: Java CI
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Clean
+        run: ./gradlew clean
+      - name: Assemble
+        run: ./gradlew assemble
+      - name: Install
+        run: ./gradlew install
+      - name: Upload
+        run: ./gradlew bintrayUpload
```

Comment:
Will this work for whole project (all modules)? I assume so, but just want to be sure

Reply:
I think it should work 🕵️‍♂️ we will see :D